src/main/java/com/guysfromusa/carsgame/repositories/GameRepository.java

```diff
@@ -9,4 +9,7 @@
  */
 @Repository
 public interface GameRepository extends JpaRepository<GameEntity, Long> {
+
+    GameEntity findByName(String name);
```

Comment:
I've created the same with optional<GameEntity> not sure if merged already (could be)